amy/workshops/tests/test_person.py

```diff
@@ -1384,7 +1384,18 @@ def setUp(self):
             password="pass",
         )
         self.user.data_privacy_agreement = True
+        self.user.github = "user-github"
+        self.user.is_active = True
         self.user.save()
+        self.user.synchronize_usersocialauth()
+        assert self.user.social_auth.all()
+
+        # Add social auth to admin
+        self.admin.github = "admin-github"
+        self.admin.is_active = True
+        self.admin.synchronize_usersocialauth()
```

Comment:
Does admin account in tests require sync with GitHub?

Reply:
Not necessarily. I mostly have just done it in case there was any difference between the way the admin works in comparison to the normal user.